ui/components/multichain/notifications-tag-counter/index.scss

```diff
@@ -2,7 +2,12 @@
   padding-left: 6px;
   padding-right: 6px;
 
+  &__text {
+    min-width: 13px;
```

Comment:
As discussed, I'm happy with min-width since the text/container can still grow.
This matches our figma.

(we can refactor in the future, this is not a big deal)